tests/test_global.py

```diff
@@ -0,0 +1,292 @@
+
+
+def test_re_ndvi_pneo():
+    # Test re_ndvi on PNEO coepelduynen and see if gives the best results.
+    path_geojson = "C:/repos/satellite-images-nso/input_data/Test_region.geojson"
+    output_path = "E:/data/test"
+
+    georegion = nso.nso_georegion(
+        path_to_geojson=path_geojson,
+        output_folder=output_path,
+        username=nso_username,
+        password=nso_password,
+    )
+
+    filepath = georegion.execute_link(
+        "https://api.satellietdataportaal.nl/v1/download/30cm_RGBNED_12bit_PNEO/20230513_104139_PNEO-03_1_1",
+        add_red_edge_ndvi_band=True,
+        plot=False,
+    )
+
+    # Open the TIFF image and display ndvi
+    with rasterio.open(filepath) as src:
+        # Get the number of bands
+        num_bands = src.count
+        print("Number of bands:", num_bands)
+
+        assert num_bands >= 7, "re_ndvi bands not added correctly"
+
+
+def test_adding_ndwi_pneo():
+    # Add the NDWI Channel
+    filepath_ndwi = nso_manipulator.add_index_channels(
+        "E:/data/test/20230513_104139_PNEO-03_1_1_30cm_RD_12bit_RGBNED_NoordwijkAanZee_Coepelduynen_cropped_Test_region_cropped_re_ndvi.tif",
+        ["ndwi"],
+    )
+
+    # Open the TIFF image and display ndvi
+    with rasterio.open(filepath_ndwi) as src:
+        # Get the number of bands
+        num_bands = src.count
+        print("Number of bands:", num_bands)
+
+        assert num_bands >= 8, "ndwi bands not added correctly"
+
+    os.remove(filepath_ndwi)
+
+
+filepath = ""
+
+
+def test_adding_ndvi_superview():
+    filepath = georegion.execute_link(
+        "https://api.satellietdataportaal.nl/v1/download/SV_RD_11bit_RGBI_50cm/20191202_110523_SV1-04",
+        add_ndvi_band=True,
+        plot=False,
+    )
+
+    # Open the TIFF image and display ndvi
+    with rasterio.open(filepath) as src:
+        # Get the number of bands
+        num_bands = src.count
+        print("Number of bands:", num_bands)
+
+        assert num_bands >= 5, "ndvi bands not added correctly"
+
+
+def test_adding_ndwi_superview():
+    # Add the NDWI Channel
+    filepath_ndwi = nso_manipulator.add_index_channels(
+        "E:/data/test/20191202_110523_SV1-04_50cm_RD_11bit_RGBI_NoordwijkAanZee_Coepelduynen_cropped_Test_region_cropped_ndvi.tif",
+        ["ndwi"],
+    )
+
+    # Open the TIFF image and display ndvi
+    with rasterio.open(filepath_ndwi) as src:
+        # Get the number of bands
+        num_bands = src.count
+        print("Number of bands:", num_bands)
+
+        assert num_bands >= 6, "re_ndvi bands not added correctly"
+
+    os.remove(filepath_ndwi)
+
+
+# Test Cloud detection
+def test_cloud_detection():
+
+    path_to_model = pickle.load(
+        open(
+            "C:/repos/satellite-images-nso/models/cloud_detection_logistic_regression_v1.0.sav",
+            "rb",
+        )
+    )
+    cloud_pictures = [
+        "E:/data/coepelduynen/20220514_114854_SV1-02_SV_RD_11bit_RGBI_50cm_Rijnsburg_natura2000_coepelduynen_cropped.tif",
```

Comment:
How does one retrieve these files?

Reply:
sadly only can be check locally

Reply:
What? That was not my question. How do I get these files, so I can run the test?

Reply:
you don't this can only currently be run locally on my pc.
TODO: is making smalle files of these so they can be shared

Reply:
You can't add these files to the public facing blob storage? If you can only run these tests on your PC they are pretty useless to anyone else.